### PR TITLE
Italian translation

### LINKED ISSUE #554: Italian translation
Hello, first of all thanks for this awesome deck!
I have noticed that an Italian translation is not available yet, I am starting to work on it right now.

### PR DESCRIPTION
**Pull request by giovanni-cutri**
Fix #554

### CODE REVIEW

File: src/data/capital_info.csv [hunk 1]
```diff
@@ -6,7 +6,7 @@ Montenegro,Cetinje is an honorary capital.,Cetinje ist eine Ehrenhauptstadt.,Cet
...
 Ukraine,Also known as Kiev.,,,,Også kjent som Kyiv.,,,,,,"基辅在国际上公认有2种名字(Kyiv, Kiev)。",,
-Kazakhstan,Known as Astana until March 2019.,Bis März 2019: Astana.,Conocida como Astaná hasta marzo de 2019.,Connue sous le nom d'Astana jusqu'en mars 2019.,Kjent som Astana til mars 2019.,Známý jako Astana do března 2019.,Известен как Астана до марта     2019 года.,Gekend als Astana tot maart 2019.,Hette Astana fram till mars 2019.,Conhecida como Astana até março de 2019.,2019 年 3 月之前称为阿斯塔纳(Astana)。,Do marca 2019 roku znany jako Astana,Conosciuta come Astana fino al marzo 2019.
+Kazakhstan,Known as Astana until March 2019.,Bis März 2019: Astana.,Conocida como Astaná hasta marzo de 2019.,Connue sous le nom d'Astana jusqu'en mars 2019.,Kjent som Astana til mars 2019.,Známý jako Astana do března 2019.,Известен как Астана до марта 2019 года.,Gekend als Astana tot maart 2019.,Hette Astana fram till mars 2019.,Conhecida como Astana até março de 2019.,2019 年 3 月之前称为阿斯塔纳(Astana)。,Do marca 2019 roku znany jako Astana,Conosciuta come Astana fino al marzo 2019.
```
Comment (aplaice) [inline src/data/capital_info.csv:9]: I think you've accidentally removed the full stop at the end of the Polish capital info for Kazakhstan.

```suggestion
Kazakhstan,Known as Astana until March 2019.,Bis März 2019: Astana.,Conocida como Astaná hasta marzo de 2019.,Connue sous le nom d'Astana jusqu'en mars 2019.,Kjent som Astana til mars 2019.,Známý jako Astana do března 2019.,Известен как Астана до марта 2019 года.,Gekend als Astana tot maart 2019.,Hette Astana fram till mars 2019.,Conhecida como Astana até março de 2019.,2019 年 3 月之前称为阿斯塔纳(Astana)。,Do marca 2019 roku znany jako Astana.,Conosciuta come Astana fino al marzo 2019.
```

File: src/data/capital_info.csv [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
...c Republic,"While Laayoune, also known as El Aaiún, is the declared capital, Tifariti is the de facto seat of government.","El Aaiún ist die erklärte Hauptstadt, aber Tifariti ist der Regierungssitz.","Aunque El Aaiún es la capital proclamada, Tifariti es la capital de facto.","Bien que Laâyoune soit la capitale déclarée, Tifariti est le siège du gouvernement.",,"Zatímco Al-´ Ajún, známý též jako Laayoune, je oficiální hlavní město, Tifariti je de facto sídlem vlády.","В то время как Лааюн, также известный как Эль-Аюн, является объявленной столицей, Тифарити является фактическим местом пребывания правительства.","Ondanks Laayoune, ook wel bekend als al-Ajoen, de verklaarde hoofdstad is, is Tifariti de feitelijke zetel van de regering.","Stavas även al-'Ayun, El Aaiún; Tifariti är de facto säte för parlamentet.","Embora Laiune, também conhecida como El Aiune, seja a capital declarada, Tifariti é a sede do governo.",虽然阿尤恩(Laayoune，西班牙语称为El Aaiún，中文含义是泉水)，是官方宣称的首都，但提法里提(Tifariti)是实际政府所在地。,"Oficjalną stolicą jest Al-Ujun, natomiast de facto funkcję stolicy pełni Tifariti.","Nonostante Laâyoune, conosciuta anche come El Aaiún, sia la capitale proclamata, Tifariti è la sede del governo."
```
Comment (aplaice) [inline src/data/capital_info.csv:17]: Nitpick: there's a trailing whitespace at the end of the capital info for SADR.

```suggestion
Sahrawi Arab Democratic Republic,"While Laayoune, also known as El Aaiún, is the declared capital, Tifariti is the de facto seat of government.","El Aaiún ist die erklärte Hauptstadt, aber Tifariti ist der Regierungssitz.","Aunque El Aaiún es la capital proclamada, Tifariti es la capital de facto.","Bien que Laâyoune soit la capitale déclarée, Tifariti est le siège du gouvernement.",,"Zatímco Al-´ Ajún, známý též jako Laayoune, je oficiální hlavní město, Tifariti je de facto sídlem vlády.","В то время как Лааюн, также известный как Эль-Аюн, является объявленной столицей, Тифарити является фактическим местом пребывания правительства.","Ondanks Laayoune, ook wel bekend als al-Ajoen, de verklaarde hoofdstad is, is Tifariti de feitelijke zetel van de regering.","Stavas även al-'Ayun, El Aaiún; Tifariti är de facto säte för parlamentet.","Embora Laiune, também conhecida como El Aiune, seja a capital declarada, Tifariti é a sede do governo.",虽然阿尤恩(Laayoune，西班牙语称为El Aaiún，中文含义是泉水)，是官方宣称的首都，但提法里提(Tifariti)是实际政府所在地。,"Oficjalną stolicą jest Al-Ujun, natomiast de facto funkcję stolicy pełni Tifariti.","Nonostante Laâyoune, conosciuta anche come El Aaiún, sia la capitale proclamata, Tifariti è la sede del governo."
```

File: src/data/country.csv [hunk 3]
```diff
@@ -1,321 +1,321 @@
...
+Anguilla,Anguilla,Anguila,Anguilla,Anguilla,Anguilla,Ангилья,Anguilla,Anguilla,Anguila,安圭拉(Anguilla),Anguilla,Anguilla
+Azores,Azoren (Habichtsinseln),Azores,Açores,Asorene,Azory,Азорские острова,Azoren,Azorerna,Açores,亚速尔群岛(Azores),Azory,Azzorre
+Curaçao,Curaçao,Curazao,Curaçao,Curaçao,Curaçao,Кюрасао,Curaçao,Curaçao,Curaçao,库拉索(Curaçao),Curaçao,Curaçao
+French Guiana,Französisch-Guayana,Guayana Francesa,Guyane,Fransk Guyana,Francouzská Guyana,Французская Гвиана,Frans-Guyana,Franska Guyana,Guiana Francesa,法属圭亚那(French Guiana),Gujana Francuska,Guyana francese
+Gibraltar,Gibraltar,Gibraltar,Gibraltar,Gibraltar,Gibraltar,Гибралтар,Gibraltar,Gibraltar,Gibraltar,直布罗陀(Gibraltar),Gibraltar,Gibilterra
+Guernsey,Guernsey,Guernsey,Guernesey,Guernsey,Guernsey,Гернси,Guersney,Guernsey,Guernesei,根西岛(Guernsey),Guernsey,Guernsey
+Isle of Man,Isle of Man,Isla de Man,Île de Man,Man,Man,Остров Мэн,Man,Isle of Man,Ilha de Man,马恩岛(Isle of Man),Wyspa Man,Isola di Man
+Jersey,Jersey,Jersey,Jersey,Jersey,Jersey,Джерси,Jersey,Jersey,Jérsia,泽西岛(Jersey),Jersey,Baliato di Jersey
```
Comment (aplaice) [inline src/data/country.csv:218]: I _think_ that this should be just "Jersey" (but I'm not 100% sure).  The part "Baliato di" (Bailiwick of) I think falls under the category of [parenthetical used for disambiguation (even if it's not literally in parentheses) and hence should probably be omitted](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#country-field).

File: src/data/country_info.csv [hunk 4]
```diff
@@ -1,111 +1,111 @@
...
+Greenland,Constituent country in the Kingdom of Denmark.,Autonomer Bestandteil des Königreichs Dänemark.,Nación constitutiva del Reino de Dinamarca.,Pays constitutif du royaume du Danemark.,Land som utgjør en del av kongeriket Danmark.,Autonomní součást Dánského království.,"Страна, входящая в состав Королевства Дании.",Autonoom gebied binnen het Koninkrijk Denemarken.,Riksdel av Danmark.,País constituinte do Reino da Dinamarca.,丹麦属地之一。,Autonomiczne terytorium zależne Danii.,Nazione costitutiva del Regno di Danimarca.
+Belarus,,Auch Weißrussland.,,,,,,Ook bekend als Belarus.,,,,,
+Czech Republic,Also known as Czechia.,,También conocida como Chequia.,Aussi connue comme la  République tchèque.,,Používané časteji Česká republika.,Известна также как Чехия.,,,,也被称为捷克(Czechia)。,Conosciuta anche come Cechia.
```
Comment (aplaice) [inline src/data/country_info.csv:9]: I think that there was a missing comma (in our current version) for the Polish translation (oops!), so the Italian country info is shifted one field to the left (there should be an extra comma between the Chinese and Italian country infos):

```suggestion
Czech Republic,Also known as Czechia.,,También conocida como Chequia.,Aussi connue comme la  République tchèque.,,Používané časteji Česká republika.,Известна также как Чехия.,,,,也被称为捷克(Czechia)。,,Conosciuta anche come Cechia.
```

File: src/data/country_info.csv [hunk 5]
```diff
@@ -1,111 +1,111 @@
...
+Faroe Islands,Constituent country in the Kingdom of Denmark.,Autonomer Bestandteil des Königreichs Dänemark.,Nación constitutiva del Reino de Dinamarca.,Pays constitutif du royaume du Danemark.,Land som utgjør en del av kongeriket Danmark.,Konstituční země Dánského království.,"Страна, входящая в состав Королевства Дании.",Autonoom gebied binnen het Koninkrijk Denemarken.,Riksdel av Danmark.,País constituinte do Reino da Dinamarca.,丹麦属地之一。,Autonomiczne terytorium zależne Danii.,Nazione costitutiva del Regno di Danimarca.
```
Comment (aplaice) [inline src/data/country_info.csv:10]: Maybe also mention one of the alternative names (`Isole Faroe`)?  (This obviously just a suggestion.)

**Review by aplaice — APPROVED** [review]
This looks fantastic (well, to the extent that I understand Italian :))!  Thanks very much!

I've made a couple of very tiny (mainly formatting) nitpicks and suggestions.

(BTW thanks also for fixing some of the formatting for some of the other languages!)

File: src/data/country.csv (hunk 3 above)
Comment (giovanni-cutri) [reply src/data/country.csv:218]: Yeah I was unsure about that, too. It's probably better to leave only "Jersey".

**Comment by aplaice** [comment]
Thanks for the fast changes!

I'll leave this open for a little while longer in case anybody has some further comments, but AFAICT it's ready for merging asap! :)

**Review by axelboc — APPROVED** [review]
Brilliant! 🤩 It all looks good to me as well.

**Pull request merged**